collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -150,8 +155,17 @@ private void setupDagger() {
         applicationComponent = DaggerAppDependencyComponent.builder()
                 .application(this)
                 .build();
-
         applicationComponent.inject(this);
+
+        audioRecorderDependencyComponent = DaggerAudioRecorderDependencyComponent.builder()
```

Comment:
This is where we can override default module dependencies if we want to (using the `dependencyModule` method of the builder). For `projects` this would let us pass in a `ProjectsRepository` for instance (letting us share access to the projects with the module).